etc/openqa/openqa.ini

```diff
@@ -157,6 +157,14 @@ blocklist = job_grab job_done
 #result_storage_duration = 365
 #important_result_storage_duration = 0
 
+[minion_task_triggers]
+# Specify one or more task names, by default these are not enabled.
```

Comment:
Please specify how "more task names" should be put here, white-space separated, comma, separated, etc. I suggest to also extend the example in L166 to have at least two tasks so that it's obvious how multiple values should be specified.

Reply:
I also added a test to `config.t`. Seems like we need special handling for space-separated keys (which I also added for this key)